tests/data/layout_invalid1.json

```diff
@@ -7,13 +7,22 @@
         "path": "derivatives"
     },
     "dpath_sourcedata": {
+        "path": "scratch"
+    },
+    "dpath_src_tabular": {
+        "path": "scratch/tabular"
+    },
+    "dpath_src_imaging": {
+        "path": "scratch/imaging"
+    },
```

Comment:
Are these incorrect because this is an invalid layout?

Reply:
No, the reason the layout is invalid is that there is an extra field `"extra_field"`.

The paths don't have to math those of the default layout since this is just testing the code.